de-lib/src/main/java/org/iplantc/de/apps/client/events/tools/ShareToolsSelected.java

```diff
@@ -23,10 +19,9 @@
     }
 
     public static final Type<ShareToolsSelectedHandler> TYPE = new Type<>();
-    private final List<App> selectedTools;
 
-    public ShareToolsSelected(List<App> selectedTools) {
-        this.selectedTools = selectedTools;
+    public ShareToolsSelected() {
```

Comment:
I already see some of the stuff I've commented on in previous PRs are getting resolved in later PRs 😄